This Month in Solid June – DO NO MERGE UNTIL JUNE 10 (#588)
solid/solidproject.org: _posts/newsletter/2021-06-09-this-month-in-solid.md

---
layout: newsletter

> title: This Month in Solid 2021-05-06
> permalink: /newsletter/2021-05-06

Reviewer: And looks like the filename should start with that too?

```suggestion
title: This Month in Solid 2021-06-10
permalink: /newsletter/2021-06-10
```

tags: [monthly, updates]
categories: [Updates]
---

### [Events](https://solidproject.org/events)

> **3 June 2021** – 03 June 2021 – Another [Solid World](https://vimeo.com/558597018) in the books! We explored how Solid reorients data to be human-centric with intersecting ideas like verifiable credentials, privacy-by-design, data reuse, and consent. 

Reviewer 2: ```suggestion
**3 June 2021** – Solid World June – Another [Solid World](https://vimeo.com/558597018) in the books! We explored how Solid reorients data to be human-centric with intersecting ideas like verifiable credentials, privacy-by-design, data reuse, and consent. 
```

Reviewer 2: Replacing the data duplicate with the name of the event you are summarizing

Solid World June kicked off with Tim Berners-Lee giving us updates about the last few weeks. One of the highlights was [his in-person visit to the Flemish Government](https://www.tijd.be/app/carousel1/tim-berners-lee-de-bedenker-van-het-internet-we-kunnen-het-wereldwijde-web-nog-fiksen/10309722.html). Right now, the Flanders Government wants to give everyone in Flanders a Solid Pod. People in Flanders (and all-around) feel a sense of excitement from seeing all the possibilities Solid has to offer and the promise of seeing one another physically. He expressed his hopes that others in the community soon get the opportunity to collaborate and feel the vibrancy of our community in person. 

> Next, we heard from Ruben Verborgh. Who broke down the need to know about verifiable credentials (VC). His focus was on connecting the dots between how VC is a standard from the W3C and how it affects Solid Pods and gives you consent back. 

Reviewer 2: ```suggestion
Next, we heard from Ruben Verborg, who broke down the need to know about verifiable credentials (VC). His focus was on connecting the dots between how VC is a standard from the W3C and how it can be used to confirm consent to access data stored in Solid Pods. 
```

> Oz Olivo, the VP of Product at Inrupt, was the final Solid roadmap update. He discussed the next steps Inrupt planning for their version 1.2 release, some of the features fell in line with the theme of this month. 

Reviewer: A bit weird to say that Oz is an update :P

```suggestion
Oz Olivo, the VP of Product at Inrupt, completed the Solid roadmap update. He discussed the next steps Inrupt planning for their version 1.2 release, some of the features fell in line with the theme of this month. 
```

> Oz Olivo, the VP of Product at Inrupt, was the final Solid roadmap update. He discussed the next steps Inrupt planning for their version 1.2 release, some of the features fell in line with the theme of this month. 

Reviewer 2: ```suggestion
Oz Olivo, the VP of Product at Inrupt, was the final Solid roadmap update. He discussed the next steps Inrupt is planning for their version 1.2 release, some of the features fell in line with the theme of this month. 
```

Reviewer 2: This edit is suggested _on top of_ Vincent's suggestion. I'm just not sure how to add a further edit to his edit.

> Our first presenter, Eliott Behar, focused on privacy, Solid, and law converge for the better. He explained how Solid not only meets the standards of General Data Protection Regulation (GDPR) but exceeds them. Eliott went over the six lawful bases that make up the GDPR described, but the most questions are the sixth, third-party processors. Solid makes it possible to see and give permissions to share data with these third parties, but how will the community design this?

Reviewer: ```suggestion
Our first presenter, Eliott Behar, focused on how privacy, Solid, and law converge for the better. He explained how Solid not only meets the standards of General Data Protection Regulation (GDPR) but exceeds them. Eliott went over the six lawful bases that the GDPR describes, but the most questions surround the sixth, third-party processors. Solid makes it possible to see and give permissions to share data with these third parties, but how will the community design this?
```

Additionally, it currently says "the sixth, third-party processors", but if I recall it correctly, the sixth lawful base was "legitimate interest", and the unclarity is about what constitutes a legitimate interest. But I'm not sure...

> Our first presenter, Eliott Behar, focused on privacy, Solid, and law converge for the better. He explained how Solid not only meets the standards of General Data Protection Regulation (GDPR) but exceeds them. Eliott went over the six lawful bases that make up the GDPR described, but the most questions are the sixth, third-party processors. Solid makes it possible to see and give permissions to share data with these third parties, but how will the community design this?

Reviewer 2: ```suggestion
Our first presenter, Eliott Behar, focused on how privacy, Solid, and law converge for the better. He explained how Solid can allow organizations to not only meet the standards of General Data Protection Regulation (GDPR), but to exceed them. Eliott went over the six lawful bases that make up the GDPR described, but the most questions are the sixth, third-party processors. Solid makes it possible to see and give permissions to share data with these third parties, but how will the community design this?
```

Tom Haegemans from [Digita](https://www.digita.ai/) presented data sharing and reuse patterns and the tools to use when thinking about its legality. Tom gave the use case of banks and governments and how we reuse data. When using Solid, we see a new way of accessing this data, like continuous access to different parties. [Digita looked at the patterns](digita.ai/resources) created when figuring out reusing data and who’s involved and tracking each user journey. In the end, we saw how there’s a pattern framework we can use when thinking about the legal considerations. 

> Last but not least, Greg Golding and Anton Wiklund, from the [Swedish Public Employment Service](https://www.digg.se/om-oss/nyheter/2021/vardefullt-for-individen-att-fa-okad-insyn-och-kontroll-over-sin-data) showed us how consent could work using Solid. Their [Proof of Concept (PoC)](https://gitlab.com/arbetsformedlingen/individdata/rupoc) was about a job seeker giving and reckoning access to their application to a prospective employer. The work happening around compliance, consent, data reuse, privacy, and law shows promise for the web being a place where individuals have true control over their data and information. 

Reviewer: Might be because I'm not a native speaker, but I don't understand what "reckoning" is supposed to mean here? Should it be requesting? Retracting?

```suggestion
Last but not least, Greg Golding and Anton Wiklund, from the [Swedish Public Employment Service](https://www.digg.se/om-oss/nyheter/2021/vardefullt-for-individen-att-fa-okad-insyn-och-kontroll-over-sin-data) showed us how consent could work using Solid. Their [Proof of Concept (PoC)](https://gitlab.com/arbetsformedlingen/individdata/rupoc) was about a job seeker giving and reckoning access to their application to a prospective employer. The work happening around compliance, consent, data reuse, privacy, and law shows promise for the web being a place where individuals have true control over their data and information. 
```

Reviewer 2: Pretty sure it should be "revoking"

> Each month we dive into new and valuable conversations about how Solid is bettering not only the web but your world. As our world and community begin opening up, we see our lives differently. Technology and data will never be the same, and Solid is changing it for the better. To continue this conversation, join us on the [Solid Forum](https://forum.solidproject.org/). Want to participate in these conversations and present at a future Solid World? Fill out the [Solid World speaker form](https://es1cz4pb7oi.typeform.com/to/nietD34f). Lastly, the Solid Team can address any questions, comments, or concerns at info@solidproject.org. 

Reviewer: I'm not sure I'd say "As our world and community begin opening up" here, with DEI in mind - most of the world is still quite far from opening up, unfortunately.

Also, to make the email address easily copyable:

```suggestion
Each month we dive into new and valuable conversations about how Solid is bettering not only the web but your world. As our world and community begin opening up, we see our lives differently. Technology and data will never be the same, and Solid is changing it for the better. To continue this conversation, join us on the [Solid Forum](https://forum.solidproject.org/). Want to participate in these conversations and present at a future Solid World? Fill out the [Solid World speaker form](https://es1cz4pb7oi.typeform.com/to/nietD34f). Lastly, the Solid Team can address any questions, comments, or concerns at [info@solidproject.org](mailto:info@solidproject.org). 
```

### In Other News
* [Digital Garden Club Event](https://www.eventbrite.com/e/understory-digital-garden-club-tickets-151311208899) – Monthly Basis
* Solid Networking Event – June 24, 2021 at 10 AM Eastern Time

> * [Diversity, Equity, & Inclustion Meeting]() 8, 2021 at 11 AM ET/ 5 PM CEST 

Reviewer: ```suggestion
* [Diversity, Equity, & Inclustion Meeting]() July 8, 2021 at 11 AM ET/ 5 PM CEST 
```

There's also a missing link here?
